### ISSUE TITLE
OracleBI Clickhouse connection with odbc driver string datatype problem

### ISSUE THREAD

**Issue by ufukaslan86** [ClickHouse/clickhouse-odbc#389]
hi,

we want to connect oracle bi 12c 2.1.4.0 version to clickhouse using odbc driver. We did configurations and connected to clickhouse default db with oracle bi. we fetch data types of numbers and dates but we can't fetch string data types. 

clickhouse-odbc-1.1.10-Linux.tar.gz
libclickhouseodbc.so

odbc.ini file info 
--------------------------------
[ClickHouse]
Driver=/u01/middleware/bi/modules/oracle.bi.datadirect.odbc/7.1.6/lib/libclickhouseodbc.so
# Optional settings:
Description = ClickHouse
server = 10.0.157.81
database = default
#uid = default
port = 8123
#sslmode = require
Timeout=300000
--------------------------

odbcinst.ini   file info
--------------------------------------
[ClickHouse]
Driver=/u01/middleware/bi/modules/oracle.bi.datadirect.odbc/7.1.6/lib/libclickhouseodbc.so
Setup=/u01/middleware/bi/modules/oracle.bi.datadirect.odbc/7.1.6/lib/libclickhouseodbc.so
----------------------------------------

Sample Analyses with date and numeric data type columns
![sampleDateNumber](https://user-images.githubusercontent.com/98466433/151171460-a778584f-233b-4cb0-ad62-e65c1eb5fff4.png)

Sample Analyses with string and numeric data type columns, looks no value it doesn't gives any error.
![stringDataType](https://user-images.githubusercontent.com/98466433/151171463-82ca6e38-a61e-4db4-a34a-309e57801d36.png)

With dbeaver screenshot. 
![stringColumnValues](https://user-images.githubusercontent.com/98466433/151172142-3fa487b1-69c4-4feb-ac07-c8c2873623b5.png)

**Comment by traceon** [comment]
Could you please enable [driver tracing](https://github.com/ClickHouse/clickhouse-odbc#troubleshooting-driver-manager-tracing-and-driver-logging) and post traces separately for the scenario with `Sample Analyses with string and numeric data type columns` and `with dbeaver`.

**Comment by ufukaslan86** [comment]
[odbctrace10.zip](https://github.com/ClickHouse/clickhouse-odbc/files/7970381/odbctrace10.zip)

this is obi's trace file. dbeaver users jdbc connection and has no error.

**Comment by ufukaslan86** [comment]
do you still want me to attach dbeaver 's  trace file.

**Comment by Enmk** [comment]
Hi @ufukaslan86 ! Were you able to resolve the issue?

**Comment by ufukaslan86** [comment]
nope. issue is continue. we found a workaround with using fixedstring datatype. but this is not our answer.
